__tests__/example.test.ts

```diff
@@ -0,0 +1,28 @@
+'use-strict';
+import * as testlib from '../src/validate';
+import * as testreq from '../src/requests';
+
+const {smarthome} = require('actions-on-google');
+const app = smarthome();
+// Test passes
+test('Test 1: Sync Response passes', async () => {
+  const exampleSyncRes = require('./example.sync.response.json');
+  const noErrors = testlib.validate(exampleSyncRes, 'sync');
+  expect(noErrors).toBe(undefined);
+});
+
+// Test fails and shows errors
+test('Test 2: Sync Response fails', async () => {
+  const exampleSyncResFail = require('./example.sync.response.fail.json');
+  const syncErrors = testlib.validate(exampleSyncResFail, 'sync');
+  console.log(syncErrors);
+  expect(syncErrors && syncErrors.join('\n')).toBe(undefined);
```

Comment:
Wouldn't we expect `syncErrors` not to be `undefined`, or for it to have a certain length?

Reply:
Yes, it shouldn't be undefined. However, I laid out the test this way so that the it throws the error of what the test expected vs. received.  Would it be clearer if I used the falsy method specified above, (in the jest format)?

Reply:
Maybe I'm not quite following how `syncErrors && syncErrors.join('\n')` is going to be `undefined`.

Reply:
From what I'm thinking the main point of this is that the syncErrors is not going to be undefined (where undefined means that the test didn't return any errors from AJV). My thinking was to write a statement that I knew would return an error - since the api response (with errors) we received is not going to what we expect (we expect no errors). Thinking about it more, if the developer wanted to make this a test that does not pass maybe they would use the falsy method instead so I'll create a test using that.